client/jetbrains/src/main/java/com/sourcegraph/config/NotificationActivity.java

```diff
@@ -0,0 +1,50 @@
+package com.sourcegraph.config;
+
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationType;
+import com.intellij.notification.Notifications;
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.options.ShowSettingsUtil;
+import com.intellij.openapi.project.DumbAwareAction;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.startup.StartupActivity;
+import com.sourcegraph.Icons;
+import org.jetbrains.annotations.NotNull;
+
+public class NotificationActivity implements StartupActivity.DumbAware {
+    @Override
+    public void runActivity(@NotNull Project project) {
+        String url = ConfigUtil.getSourcegraphUrl(project);
+        if (ConfigUtil.isUrlNotificationDismissed() || (url.length() != 0 && !url.startsWith("https://sourcegraph.com"))) {
+            return;
+        }
+        // Display notification
+        Notification notification = new Notification("Sourcegraph", "Sourcegraph",
+            "A custom Sourcegraph URL is not set for this project. You can only access public repos. Do you want to set your custom URL?", NotificationType.INFORMATION);
+        AnAction setUrlAction = new DumbAwareAction("Set URL") {
+            @Override
+            public void actionPerformed(@NotNull AnActionEvent anActionEvent) {
+                ShowSettingsUtil.getInstance().showSettingsDialog(project, SettingsConfigurable.class);
+            }
+        };
+        AnAction cancelAction = new DumbAwareAction("Do not Set") {
```

Comment:
Should this be

```suggestion
        AnAction cancelAction = new DumbAwareAction("Do not set") {
```

or 

```suggestion
        AnAction cancelAction = new DumbAwareAction("Do Not Set") {
```

?

The other button is in title case hence my confusion.

Reply:
Oh, dang, I meant to set this to title case, just I thought "not" was not capitalized in title case. How embarrassing. Thanks for the fix! :)